meta/main.yml

```diff
@@ -0,0 +1,30 @@
+galaxy_info:
+  author: Team Starship
```

Comment:
I think Author should reflect the original author, as we made no changes to the actual code (same for company, just dropping that?)